demo/Makefile

```diff
@@ -74,7 +74,7 @@ token:
 
 # Open dashboard in browser
 dashboard:
-	@open http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token}
+	@URL=http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token} && xdg-open $$URL || open $$URL
```

Comment:
It would be cleaner to have a variable like `$(OPEN)` which is set conditionally. I don't know the exact syntax, but something like:

```
OPEN := open
if $(which xdg-open)
  OPEN = xdg-open
endif
```